src/Psd/PsdNativeFile_Linux.cpp

```diff
@@ -0,0 +1,164 @@
+
+#include "PsdPch.h"
+
+#include "PsdLog.h"
+#include "PsdFile.h"
+#include "PsdAllocator.h"
+#include "PsdNamespace.h"
+#include "PsdMemoryUtil.h"
+#include "PsdNativeFile_Linux.h"
+
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <unistd.h>
+#include <fcntl.h>
+#include <aio.h>
+
+#include <cstring>
+#include <cstdlib>
+#include <cwchar>
+#include <string>
+
+
+
+PSD_NAMESPACE_BEGIN
+/**
+ * @brief Convert wchar_t * to char *
+ * 
+ * @param ws 
+ * @return char * 
+ */
+static char *convert_ws(const wchar_t *ws){
+    char *buffer;
+    size_t n = std::wcslen(ws) * 4 + 1;
+    buffer = static_cast<char*>(std::malloc(n));
+    std::memset(buffer,0,n);
+    if(buffer == nullptr){
+        return nullptr;
+    }
+    std::wcstombs(buffer,ws,n);
+    return buffer;
+}
```

Comment:
I think it would be much nicer if such helper functions go into a separate file, e.g. Util\PsdStringUtil or similar?